packages/main/src/Calendar.ts

```diff
@@ -319,6 +330,32 @@ class Calendar extends CalendarPart {
 		this.secondaryCalendarType && this._setSecondaryCalendarTypeButtonText();
 	}
 
+	extractDateModes():string[] {
```

Comment:
4. Use the syntax of `Array<T>` instead of `T[]`
as described in https://github.com/SAP/ui5-webcomponents/blob/main/docs/5-development/07-ts.md